src/components/sbb-pearl-chain-time/sbb-pearl-chain-time.scss

```diff
@@ -41,6 +41,10 @@
   margin-inline-start: calc(var(--sbb-spacing-fixed-2x) - sbb.px-to-rem-build(4));
 }
 
+.sbb-pearl-chain__time-walktime-prime-symbol {
```

Comment:
This change confuses me a little bit, is this an improvement?

Reply:
As I had also pointed out to Tom in the PR of the notification, with Lit we have a different handling of spaces with inline elements so we have to be careful where we have `&nbsp;` or spans side by side because otherwise we would have more space than we should between the elements. These styles I have added have served to adjust those spaces.